app/src/lib/ui/Manager/ImportBeneficiaries.svelte

```diff
@@ -138,6 +132,18 @@
 		});
 	}
 
+	async function importBeneficiariesApi(beneficiaries) {
```

Comment:
Je propose de renommer la fonction pour ne pas laisser le détail technique "transpirer".

```suggestion
	async function importBeneficiaries(beneficiaries) {
```